src/targets/anvil.js

```diff
@@ -0,0 +1,104 @@
+import { ClickableObject } from "../clickable-object.js";
+import { CONSTANTS } from "../constants/constants.js";
+import { characterData } from "../cookie-io.js";
+import { Button } from "../ui/button.js";
+import { getItemClass, hasItem } from "../utilities.js";
+
+export class Anvil extends ClickableObject {
+    name = "Anvil";
+    varName = "bronzeDagger"; // Use # bronze daggers forged as quest
+    examineText = "Used for fashioning metal items.";
+    actions = [
+        { text: "Forge", func: "clickTarget" },
+        { text: "Examine", func: "examine" },
+    ];
+
+    constructor(scene) {
+        super();
+        this.scene = scene;
+
+        const cameraWidth = scene.cameras.main.width;
+        const cameraHeight = scene.cameras.main.height;
+        const width = 220;
+        const height = 140;
+        const x = cameraWidth / 2 - width + 60;
+        const y = cameraHeight / 2 - height + 40;
+
+        // Add invisible button for anvil
+        this.sprite = new Button(scene, x, y, width, height);
+        this.sprite.on("pointerdown", (pointer) => {
+            if (pointer.rightButtonReleased() && !pointer.leftButtonDown()) {
```

Comment:
Right click menu doesn't seem to work, and just realized it doesn't work on the furnace level. Not sure what's wrong there @CurtisGreen

Reply:
Just checked it out, is there a reason it's `rightButtonReleased()`? It works as expected with `rightButtonDown`

Reply:
I may have accidently introduced that when fixing other right click bugs. It should be rightButtonDown

Reply:
Alrighty, Andy when you swap this one can you swap it on furnace.js as well?

Reply:
Yes I will